recipes/ipopt/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "Ipopt" %}
+{% set version = "3.12.6" %}
+{% set sha256 = "b1b04fb2aaf424464c00c939fdd0b4075614d8d689b7bbba3cbaedf4a4c997a5" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.coin-or.org/download/pkgsource/{{ name }}/{{ name }}-{{ version }}.tgz
+  sha256: {{ sha256 }}
+  patches:
+    - mumps_mpi.patch
+
+build:
+  skip: True  # [win]
+  number: 0
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - ampl-mp
+    - mumps
+    - metis
+    - scotch
+    - gcc
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.19|0.2.19.*
+
+  run:
+    - ampl-mp
+    - mumps
+    - metis
+    - scotch
+    - gcc
```

Comment:
I'm assuming we need the whole compiler, mainly because the test compiles a test program and makes sure the solution is optimal. I could cut these though if that's not warranted.
```bash
g++ -I$PREFIX/include/coin  -c -o cpp_example.o cpp_example.cpp
...
./cpp_example | grep -q "Optimal Solution"
```